lib/utils/test_utils.go

```diff
@@ -36,7 +36,8 @@ func NewTestDir(t *testing.T) string {
 func NewTestBasePath(t *testing.T, name string) string {
 	t.Helper()
 
-	basepathTmp := path.Join(t.TempDir(), t.Name(), name)
+	tmpdir := t.TempDir()
+	basepathTmp := path.Join(tmpdir, t.Name(), name)
```

Comment:
Maybe just use `os.TempFile` instead?

Reply:
Actually just change to

```suggestion
	tmpdir := t.TempDir()
	basepathTmp := path.Join(tmpdir, name)
```

the test name is already in `tmpdir`